src/bittybuzz/src/BittyBuzzMathFunctions.cpp

```diff
@@ -147,7 +147,7 @@ void BittyBuzzMathFunctions::bbzmath_sin() {
         return;
     }
 
-    bbzvm_pushf(bbzfloat_tofloat(Math::sin(arg.value())));
+    bbzvm_pushf(bbzfloat_fromfloat(Math::sin(arg.value() * Math::pi / 180)));
```

Comment:
So the buzz scripts will work in degrees? Why is the conversion only done once? Shouldn't the result be converted back before pushing back to the VM?

Reply:
Since our neighbors values are in degrees, I though it would be simpler to have all the functions take degrees instead of rad. Unless we want to migrate all to rad, though we may lose a bit of precision since rads are smaller and idk how float_16 are precise. You are right though, the return value needs to be converted in degrees again.

Reply:
Yeah, you're right, I don't know what I was thinking